src/components/Modals/Send/Send.tsx

```diff
@@ -41,9 +42,13 @@ export const SendModal = ({ asset, accountId }: SendModalProps) => {
           <Switch>
             <Route
               path='/'
-              component={(props: RouteComponentProps) => (
-                <Form asset={asset} accountId={accountId} {...props} />
-              )}
+              component={(props: RouteComponentProps) => {
+                // TODO(gomes): make this cleaner
```

Comment:
can you please make a ticket for this, referring to @cjthompson discussion on things that plugins need to register?

Reply:
https://github.com/shapeshift/web/issues/1234